README.md

```diff
@@ -112,43 +172,6 @@ This list is for websites, services, software, tools and more: everything that y
 - [GC-Prevue](http://www.graphicode.com/GC-Prevue_Gerber_Viewer) - Commercial with free version. Can handle some gerbers better than Gerbv and KiCAD.
 - [ZofZPCB](https://www.zofzpcb.com/) - FREE 3D Gerber Viewer.
 
-## Free EDA Packages
-- [KiCad](https://kicad.org/) - Open source EDA package with push and shove router, differential pairs and much more.
-- [Eagle](https://www.autodesk.com/products/eagle/overview) - One of the most popular EDA packages due to it's (board size restricted) free version.
-- [DesignSpark PCB](https://www.rs-online.com/designspark/pcb-software) - Gratis EDA package without restrictions, sponserd by RS Components.
-- [Altium CircuitMaker](https://circuitmaker.com/) - Free package from the maker of the go to pro software.
-- [gEDA](http://geda-project.org) - Another open source package, good for people that like scripting and makefiles, Linux and BSD only.
```

Comment:
AFAIK, gEDA is a [dead](git.geda-project.org/geda-gaf.git) project, with almost no update and not used in practice anymore... I am not a user of this tool, though. So let me know whether it is a active project or is we should remove it from the awesome list...

Reply:
It's ok to remove it I think. Please do leave gerbv under gerber viewers though, I still use that and it works great.